a10_octavia/install/install-a10-octavia

```diff
@@ -14,6 +25,29 @@ HEALTHMANAGER="a10-health-manager.service"
 
 SERVICES=("$CONTROLLER" "$HOUSEKEEPER" "$HEALTHMANAGER")
 
+for service in ${SERVICES[@]}; do
+        if id "octavia" >/dev/null 2>&1; then
+            sed -i 's/%user%/octavia/' "${SYSTEMD_SCRIPT_DIR}/${service}"
+        else
```

Comment:
With this user check in place a second script needs to be created for the developer installation. We shouldn't try to start the service on the `stack` user as a part of this script as we can't be sure if said user is openstack related in other envs.

Reply:
Created a separate script for developer installation with user as `octavia` OR `stack`